buffer/src/main/java/io/netty/buffer/api/internal/ResourceSupport.java

```diff
@@ -90,6 +90,7 @@ public final void close() {
         if (acquires == 0) {
             tracer.drop(acquires);
             drop.drop(impl());
+            makeInaccessible();
```

Comment:
I was wondering if we should call `makeInaccessible()` in a finally block. Same comment below.